src/components/gauge/gauge.scss

```diff
@@ -0,0 +1,222 @@
+@import "../common/_base";
+
+gx-gauge {
+  /**
+  * @prop --component-height:
+   * Set the height or the control. *Must be set before render the control*
+   * (400px by default)
+  */
+  --component-height: 400px;
+
+  /**
+  * @prop --center-circle-text-color:
+   * Define the color of the center text in `circle` gauge type.
+   * (gray by default)
+  */
+  --center-circle-text-color: rgba(0, 0, 0, 0.7);
+
+  /**
+  * @prop --marker-backgorund:
+   * Define the background color of the marker.
+   * (1px by default)
+  */
+  --marker-backgorund: rgba(0, 0, 0, 0.7);
```

Comment:
Is this property intended just for setting a background color?
If so, I think it should be named `--marker-background-color`.
BTW, it seems there are other properties in the same situation.

Reply:
Yes! I'll rename some vars

Reply:
Check out the new names in commit 64c6713 !